src/_tutorials/dart-vm/cmdline.md

```diff
@@ -78,14 +78,12 @@ void main() {
 
 <li markdown="1">
 In the directory that contains the file you just created,
-run the program with the command as shown by the highlighted text.
+run the program with the command as shown next.
```

Comment:
Let's change this to:

... run the program:

(I don't think the words afterward add anything.)

Reply:
Ah, yes. Done.